### DISCUSSION TITLE
Digital Signage custom software

### DISCUSSION THREAD

**Discussion by cinideep** [Syncplay/syncplay#508] [Archive]
Hello,
First of all appreciate the wonderful development you have done.
Would like to know if we can use this for digital signage, there will be 
1. Custom made media player. From the server video files will be pushed to multiple viewers
2. Playlist will have multiple videos. All need to synchronized

Can we modify the code for this?

**Comment by eirnym** [comment]
I'd like to mention, that you're always welcome to extend sync play to support more players.

Currently  it's possible to manipulate players, providing a common modifiable [playlist](https://syncplay.pl/guide/client/) and it's manipulation, so I don't think your solution would require any additional programming beside a custom player if required. If it would, project will only benefit from new extensions if you'd create PRs to extend it.

**Comment by Et0h** [comment]
Digital signage falls outside of Syncplay's intended use case.

Syncplay is open source under the Apache-2.0 License so anyone who wants to make their own implementation under that license.
